app/src/main/java/com/example/next_show/fragments/FeedFragment.java

```diff
@@ -151,41 +168,116 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
         return currView;
     }
 
+    private void resetPage() {
+        // clears the screen and current list of shows
+        adapter.clear();
+
+        // resets filters
+        showFilter.setUpFilters();
```

Comment:
Hm -- maybe we should name this so that it's clearer what it does.  `showFilter.resetFilters()`